data_contribution_2_data.md

```diff
@@ -0,0 +1,907 @@
+
+The CHOP project template evolved through iterations and feedback from CHOP researchers. A multi-disciplinary groups of practicioners were consulted in the template adaptation development, including:
+
+- Bioinformatics
+- Cancer research
+- Microbiome center
+- Research IT
+- Clinical sequening unit
+- Medical Informatics Unit
+
+**How is the Project Template used at Arcus?**
+
+The Project Template prioritizes streamlined archiving and reproducible research pathways. It archives a wide range of research types from the Research Institute, making them discoverable through tools like [Arcus Cohort Discovery](https://arcus.chop.edu), [Gene](https://chop.alationcloud.com), and [Omics Variant Browser](https://arcus.chop.edu). The Project Template facilitates organizing diverse research data in a single directory structure, enabling automated archiving, metadata management, and data delivery throughout the research data lifecycle. This file directory structure is used for the entire lifecycle of research data within Arcus:
+
+![Graphic representation of the lifecycle of the project template in Arcus](media/project_template/ProjectTemplate_Lifecycle.png)
+
+The project template provides a shared structure so that institutional knowledge previously held locally by various members of the data creation team becomes centralized.
+
+The utility of [the project template](https://github.research.chop.edu/arcus/rdm-project-template) for lab drive organization and integration with the Arcus archives is summarized in the graphic below.
+
+![Graphic representation of the workflow of research data and the project tempalte](media/project_template/ProjectTemplate_workflow.png)
+
+## Project Template
+
+The project template structure includes directories for capturing three major aspects of a research effort: the data (data), the tools needed to work with that data (access tools), and the contextual information needed to understand the effort and its constituent parts (contextual). The high level directories are as follows (items with asterix are required):
+
+- **_Configs_** (contextual)
+- **_Data_** (data)\*
+- **_Manifests_** (data)\*
+- **_Models_** (access tools)
+- **_References_** (contextual)
+- **_Reports_** (contextual)
+- **_Requirements_** (contextual)
+- **_SRC_** (access tools)
+
+Below is an image of the entire Project Template Directory, with more detail about each section:
+
+![Graphic respresentation of the Project template with a short explanation of all the different sections](media/project_template/ProjectTemplate_Description.png)
```

Comment:
Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases.  I leave it to you to decide if the effort of re-doing is worth it.